lib/pages/chat_profile_info/chat_profile_info.dart

```diff
@@ -44,13 +57,37 @@ class ChatProfileInfoController extends State<ChatProfileInfo> {
     const Right(LookupContactsInitial()),
   );
 
+  final GlobalKey<NestedScrollViewState> nestedScrollViewState = GlobalKey();
+
+  final List<ChatDetailsPage> profileSharedPagesList = [
+    ChatDetailsPage.media,
+    ChatDetailsPage.links,
+    ChatDetailsPage.files,
+  ];
+
+  SameTypeEventsBuilderController? _mediaListController;
+  SameTypeEventsBuilderController? _linksListController;
+  SameTypeEventsBuilderController? _filesListController;
+  TabController? tabController;
+
+  Timeline? _timeline;
+
```

Comment:
IMO,  you can create a mixin `ChatDetailsTabMixin on HandleVideoDownloadMixin, PlayVideoActionMixin, SingleTickerProviderStateMixin`

Reply:
I created a mixin but if you expect things link `ChatDetailsTabMixin on HandleVideoDownloadMixin, PlayVideoActionMixin, SingleTickerProviderStateMixin`, it will lead to an error: `The class 'ChatProfileInfoController' can't implement both 'State<ChatProfileInfo>' and 'State<StatefulWidget>' because the type arguments are different.`